api4/webhook_test.go

```diff
@@ -1048,12 +968,9 @@ func TestUpdateOutgoingHook(t *testing.T) {
 		createdHook.DisplayName = "Basic user 2"
 		updatedHook, resp := Client.UpdateOutgoingWebhook(createdHook)
 		CheckNoError(t, resp)
-		if updatedHook.DisplayName != "Basic user 2" {
-			t.Fatal("should apply the change")
-		}
-		if updatedHook.CreatorId != th.SystemAdminUser.Id {
-			t.Fatal("hook creator should not be changed")
-		}
+
+		require.Exactly(t, "Basic user 2", updatedHook.DisplayName, "should apply the change")
```

Comment:
why the use of exactly here instead of equal? Not that i'm against it, but I'm not sure why are you using over the other in different sections of code?

Reply:
Used it because it expects a fixed value, not a property.